polyfill/lib/zoneddatetime.mjs

```diff
@@ -561,6 +561,16 @@ function bigIntIfAvailable(wrapper) {
   return typeof BigInt === 'undefined' ? wrapper : wrapper.value;
 }
 
+function bigIntFloorDiv(left, right) {
+  left = bigInt(left);
+  right = bigInt(right);
+  const { quotient, remainder } = left.divmod(right);
+  if (remainder != 0 && !left.isNegative() != !right.isNegative()) {
```

Comment:
If right is positive, as it always is in our code, then is this condition equivalent to `if (remainder.isNegative()) {` ? If yes then a comment explaining that could be nice here.

I didn't realize that `!= 0` worked with this bigint library. Good to know.

Reply:
The `!= 0` was a mistake — I'll fix it on Monday. (It works in this case because I accidentally wrote `!=` instead of `!==` due to having written C++ all week 😅 So the type coercion silently calls `remainder.valueOf()`, which in this particular bigint library returns a Number.)